homeassistant/components/airzone/strings.json

```diff
@@ -76,6 +76,9 @@
     "sensor": {
       "rssi": {
         "name": "RSSI"
+      },
+      "thermostat_signal": {
+        "name": "Signal percentage"
```

Comment:
Ideally we don't have the unit of measurement in the entity name, should we just call it signal strength?

Reply:
Sure, no problem

Reply:
Done in 1772366fb106d2aff61b165fc25eea5c8ffa868e